### PR TITLE
Pull request to publish the template for Beswifty.com

### PR DESCRIPTION
**Pull request by beswifty**

### CODE REVIEW

File: beswifty.com.website.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+   "providerId":"beswifty.com",
+   "providerName":"Beswifty",
+   "serviceId":"website",
+   "serviceName":"Swifty Site",
+   "version": 4,
```
Comment (pawel-kow) [inline beswifty.com.website.json:6]: Starting with 4? Or copy-paste issue?

File: beswifty.com.website.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+   "logoUrl":"https://beswifty.com/wp-content/uploads/2019/12/Swifty-Logo_Tagline.svg",
+   "description":"Enables a domain to work with Swifty Sites",
+   "syncRedirectDomain":"*.beswifty.com",
```
Comment (pawel-kow) [inline beswifty.com.website.json:9]: The specification does not foresee use of wildcards here. Multiple domains are possible though.
> When present, this is a comma separated list of domain names for which redirects must be sent to after applying a template for the synchronous flow.

File: beswifty.com.website.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      {
+         "type":"CNAME",
+         "host":"www",
+         "pointsTo":"@",
+         "ttl":600
+      },
+      {
+         "type":"A",
+         "host":"@",
+         "pointsTo":"%host%",
```
Comment (pawel-kow) [inline beswifty.com.website.json:20]: `%host%` is a reserved variable equal to `host` parameter of `/apply` call.
Also, if the IP address is variable, the template shall either use signed apply (specify syncPubKeyDomain) or be marked as insecure (warnPhishing set to true).
See https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#423-security-considerations for details

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: beswifty.com.website.json (hunk 1 above)
Comment (beswifty) [reply beswifty.com.website.json:6]: Copy Paste Issue

File: beswifty.com.website.json (hunk 3 above)
Comment (beswifty) [reply beswifty.com.website.json:21]: This version is mainly for the internal use. So we are okay setting warnPhishing to true in this version

**Pull request merged**